plugins/janus_videoroom.c

```diff
@@ -408,7 +408,7 @@ room-<unique room ID>: {
  * (although may choose not to, more on this later) publish media in the
  * room, and as such become feeds that you can subscribe to.
  *
- * To specify a handle will be associated with a publisher, you must use
+ * To specify a handle which will be associated with a publisher, you must use
```

Comment:
The "which" is wrong here: the sentence means to say that "to specify THAT a (core) handle will be associated (is going to be associated) with a publisher, you mus use...".